knip.jsonc

```diff
@@ -0,0 +1,8 @@
+{
+	"$schema": "https://unpkg.com/knip@latest/schema.json",
```

Comment:
Wouldn't it be better to use `node_modules/knip/schema-jsonc.json`, that way we're always using the right schema for the version we have installed?

Reply:
Yeah, and there's also e.g. `https://unpkg.com/knip@5/schema.json` for this. Pushed this.